sdk/url_values.go

```diff
@@ -0,0 +1,37 @@
+package sdk
+
+import (
+	"net/url"
+	"strconv"
+)
+
+type urlValues interface {
+	UrlValues() *url.Values
+}
+
+type urlValuesBuilder struct {
+	data map[string]any
+}
+
+func (u *urlValuesBuilder) UrlValues() *url.Values {
```

Comment:
I think using new struct and interface is a bit overcomplicated. Consider using just a function:
```
func toUrlValues(data map[string]any) *url.Values {
    ...
}
```
which can be used in `(*FingerprintApiGetVisitsOpts).UrlValues` directly.